docs/Reference/RPC/buildRPC_Manual.md

```diff
@@ -168,5 +168,54 @@ The toolkit provides a reference implementation called the Sample Web UI to mana
     !!! error "Troubleshooting"
         Run into an issue? Try these [troubleshooting steps](../troubleshooting.md).
          
+## <a name="TransitionDevice"></a>Transition Activated Device
 
+If an Intel vPro® Platform has been previously activated, either in the BIOS or with another management solution or tool, it can be brought under Open AMT Cloud Toolkit control with the rpc-go application. 
 
+Additionally, use the following instructions to transition from a previously established toolkit stack to a fresh installation on a new development system.
+
+!!! Note
+    Use the following instructions to transition devices to either ACM or CCM profiles. You will need the AMT password.
+    
+    Open AMT Cloud Toolkit increases security with multiple passwords. Find an explanation of toolkit passwords in [Reference -> Architecture Overview](../Reference/architectureOverview.md#Passwords).
```

Comment:
Need to remove the /Reference from the relative path to fix the build error.  Should be (../architectureOverview.md#Passwords)